src/router/index.ts

```diff
@@ -8,6 +8,7 @@ import { unreachable } from "@app/utils";
 const documentTitle = "Radicle Interface";
 
 export const historyStore = writable<Route[]>([{ resource: "home" }]);
+let navigationMethod = "history";
```

Comment:
This doesn't look right; it shouldn't be allowed to mutate this var at runtime.

Reply:
Still have some improvements to do but I think this is a better approach

https://github.com/radicle-dev/radicle-interface/commit/d5e43f3d43a0da86a13f20692717c4fe6f8c6fdd